### PR TITLE
ANGOLASUP-825: Fixed inventory saving functionality

### PR DESCRIPTION
**Pull request by sradziszewski**
Added functionality for caching products added with a new lot code.

JIRA task link: https://openlmis.atlassian.net/browse/ANGOLASUP-825

### CODE REVIEW

**Review by dariapionk — COMMENTED** [review]
when we add comments like '// ANGOLASUP-825: ' there is no need to clarify what exactly hapenned in specific scope
You can change all of commented lines to 'ANGOLASUP-825: Fixed inventory saving functionality' (same as title of your PR)
The rest looks good to me

**Comment by kmadej** [comment]
@sradziszewski please apply Daria's comment

File: src/stock-edit-lot-modal/edit-lot-modal.service.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        .module('stock-edit-lot-modal')
+        .service('editLotModalService', service);
+
+    service.$inject = ['openlmisModalService'];
+
+    function service(openlmisModalService) {
+
+        this.show = show;
+
+        // ANGOLASUP-825: Changed function parameters from item list to draft
```
Comment (kmadej) [inline src/stock-edit-lot-modal/edit-lot-modal.service.js:37]: changed this comment also

**Review by kmadej — CHANGES_REQUESTED** [review]

File: src/stock-edit-lot-modal/edit-lot-modal.service.js (hunk 1 above)
Comment (sradziszewski) [reply src/stock-edit-lot-modal/edit-lot-modal.service.js]: comment changed

**Review by dariapionk — APPROVED** [review]

**Comment by kmadej** [comment]
@sradziszewski looks good to me, @dariapionk please merge

**Pull request merged**